src/client/views/index.html

```diff
@@ -35,6 +35,8 @@
 	<script src="https://appsforoffice.microsoft.com/lib/1.1/hosted/office.js"></script>
 	<app class="app">
 		<div id="progress" class="ms-progress-component">
+			<div class="ribbon left"></div>
```

Comment:
Do we really need am empty `div`. If that's the case I feel you should be able to just do `#progress::before` & `#progress::after` instead